endor/src/routes/auth.routes.js

```diff
@@ -120,16 +126,17 @@ router.post('/oauth2/authorize', [authController.isAuthenticated].concat(authVal
  *   "token_type": "Bearer"
  * }
  */
-router.post('/oauth2/token', [authController.isClientAuthenticated].concat(authValidator.checkToken()), authController.token());
+router.post('/oauth2/token', [authController.isAuthenticated].concat(authValidator.checkToken()), authController.token());
```

Comment:
Can we change the endpoints to something other than oauth2? It's not really what we are doing. And it will be more clear to what is going on.